docs/examples/imageset-config-filter-catalog.yaml

```diff
@@ -15,7 +15,7 @@ mirror:
   ocp:
   operators:
     - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9
-      headsOnly: false
+      allPackages: true
```

Comment:
I think headsonly encompasses all packages at their channel heads, but allPackages doesnt encompass all channels within the packages. How about `full` or just `all`?

Reply:
`full` is fine with me.